scripts/InstallerYAMLSyntax.md

```diff
@@ -41,7 +41,7 @@ Item | Description | Required?
 `name:` | Block-device alias and partition number or the physical partition name| Yes
 `type:` | Partition type should be `part` for a standard partition or `crypt` for encrypted partitions | Yes
 `fstype:` | Type of the partition can be one of: `swap`, or `ext2`, `ext3`, `ext4`, `xfs`, `btrfs`, or `vfat` | Yes
-`size:` | Size of of partition. The suffixes `B` for bytes, `K` for kilobytes, `M` for megabytes, `G` for gigabytes, `T` for terabytes, or `P` for pedabytes can be used. | Yes 
+`size:` | Size of of partition. Set to `0` to use the remaining free space for this partition; there can only be one partition of size `0`.The suffixes `B` for bytes, `K` for kilobytes, `M` for megabytes, `G` for gigabytes, `T` for terabytes, or `P` for pedabytes can be used. | Yes 
```

Comment:
While at it, can you please fix the existing typos.. 
"Size of of partition.." - remove the extra "of"
"pedabytes" - should be "petabytes"